core/templates/filters/format-base-translatable-object-values.filter.spec.ts

```diff
@@ -0,0 +1,91 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Tests formatBaseTranslatableObjectValues filter.
+ */
+
+// TODO(#7222): Remove the following block of unnnecessary imports once
+// the code corresponding to the spec is upgraded to Angular 8.
+import { UpgradedServices } from 'services/UpgradedServices';
+// ^^^ This block is to be removed.
+
+require('filters/format-base-translatable-object-values.filter');
+
+const DEFAULT_OBJECT_VALUES = require('objects/object_defaults.json');
+
+describe('formatBaseTranslatableObjectValues filter', function() {
+  const filterName = 'formatBaseTranslatableObjectValues';
+
+  beforeEach(angular.mock.module('oppia'));
+
+  beforeEach(angular.mock.module('oppia', ($provide) => {
+    const upgradedServices = new UpgradedServices();
+    for (let [key, value] of Object.entries(
+      upgradedServices.getUpgradedServices())
```

Comment:
You shouldn't need Upgraded services... You can use importAllAngularServices instead.

Reply:
Fixed, thanks!